### ISSUE TITLE
\l too verbose

### ISSUE THREAD

**Issue by effbiae** [kevinlawler/kona#456]
\f evaluates each line and prints the result.  it's too verbose.  (as of date of this post)

```
K Console - Enter \ for help

  \cat f.k   
x:!10
+/x
  \l f.k
0 1 2 3 4 5 6 7 8 9
45

```
### kc.c:252
```
I line(FILE*f, S*a, I*n, PDA*p) {  //just starting or just executed: *a=*n=*p=0,  intermediate is non-zero
  S s=0; I b=0,c=0,m=0; K k; F d; fer=0;

  //I o = isatty(STDIN) && f==stdin; //display results to stdout?
  I o = isatty(STDIN); //display results to stdout?

  if(-1==(c=getline_(&s,&m,f))) GC;

```

**Comment by kirbyfan64** [comment]
#353?

**Comment by effbiae** [comment]
yes, #353 - this is complementary or a duplicate.

**Comment by tavmem** [comment]
This was fixed by commit 0e6351523ba28def719fe44f1c994764ea57ac02
